README.md

```diff
@@ -85,3 +90,14 @@ Discuss features and ask questions on
 ## License
 
 MIT
+
+## Module Long Term Support Policy
+This module adopts the [Module Long Term Support (LTS)](http://github.com/CloudNativeJS/ModuleLTS) policy, with the following End Of Life (EOL) dates:
+
+| Version | Status          | Published | EOL      |
+| ------- | --------------- | --------- | -------- |
+| 6.x     | Active LTS      | Apr 2018  | Dec 2019 |
```

Comment:
AFAICT, the latest (unpublished) version of loopback-cli is 5.x. Are there any plans to release 6.0 soon?

Please describe the following older releases as being in Maintenance LTS mode:

 - The first 4.x version was released in September 2017 (see https://github.com/strongloop/loopback-cli/commit/d704a5aebdec7f7a5cf96fff109bb7d48ba9af03#diff-b9cfc7f2cdf78a7f4b91a753d10865a2). It will reach EOL in Dec 2019.

 - The first 3.x release was released in July 2017 (see https://github.com/strongloop/loopback-cli/commit/4fcded77d48956010cd997d8fec942a72dc513ba#diff-b9cfc7f2cdf78a7f4b91a753d10865a2), 3.x will reach EOL in Apr 2019.

 - The first 2.x was released in March 2017 (see https://github.com/strongloop/loopback-cli/commit/f685730b200ac6288d68000c929ea7904f547609#diff-b9cfc7f2cdf78a7f4b91a753d10865a2), EOL will be Apr 2019.

 - The first 1.x was released in Januarry 2017 (see https://github.com/strongloop/loopback-cli/commit/f72fea5f3840e547baca643497eb17e75223abaf#diff-b9cfc7f2cdf78a7f4b91a753d10865a2), EOL will be Apr 2019.

Reply:
@bajtos 😨 good catch

Reply:
Nope we haven't planned for 6.0 yet...